### PR TITLE
Fix selection behavior for polymer 1.0

### PR DESCRIPTION
**Pull request by mdcfrancis**
Polymer 1.0 defines pages as div elements in current examples.  Virtual patch of attribute boolean values fails with sneaky dom and results in an always true value. This breaks the tabbed pages example.

### CODE REVIEW

File: src/library/layout2.jl [hunk 1]
```diff
@@ -67,7 +67,7 @@ end
 
 render(ps::Pages, state) =
     Elem("iron-pages",
-        map(t -> Elem("section", render(t, state)), ps.pages.tiles),
+        map(t -> Elem("div", render(t, state)), ps.pages.tiles),
```
Comment (shashi) [inline src/library/layout2.jl]: why was this needed?

File: src/basics/behavior.jl [hunk 2]
```diff
@@ -131,11 +131,10 @@ end
 
 render(t::Selectable, state) =
     render(t.tile, state) <<
-        Elem("selectable-behavior",
+        Elem("selectable-behavior", multi = t.multi,
             attributes = @d(
                 :name=>t.name,
-                :selector=>t.selector,
-                :multi=>t.multi
```
Comment (shashi) [inline src/basics/behavior.jl]: The right way to fix this is to change this line to `:multi=>boolattr(t.multi)` right here. moving it out from attributes will cause updates to not trigger "multi-changed" event (assuming there is one).

File: src/library/layout2.jl (hunk 1 above)
Comment (mdcfrancis) [inline src/library/layout2.jl]: It is not required, the current docs for iron-pages indicate that these should be divs. I'd assume that  selection would work, but I went with consistency. 

https://elements.polymer-project.org/elements/iron-pages

Comment (shashi) [inline src/library/layout2.jl]: ah, I guess that's fine.

**Comment by mdcfrancis** [comment]
I believe you can use the reflect to attribute on the polymer element to lift the value. But your solution looks good.

**Comment by shashi** [comment]
having properties and attributes mean different things pretty messy. We will find a nice way to fix #72. But for now, could you update the PR?

**Comment by mdcfrancis** [comment]
Sure will. Thanks for looking.
